pom.xml

```diff
@@ -67,7 +67,7 @@
         <junit.version>5.8.2</junit.version>
         <jetty.version>10.0.7</jetty.version>
         <jersey.version>2.35</jersey.version>
-        <jackson.version>2.12.6</jackson.version>
+        <jackson.version>2.13.2</jackson.version>
```

Comment:
Can you update to 2.13.3 ?

Reply:
Ok but in that case whois-internal jackson dependency also needs to be updated whenever we update whois depenedency

Reply:
OK we can do it some other time. This PR is 👍